### PR TITLE
build(deps): bump @faker-js/faker from 7.6.0 to 8.0.2

### PR DESCRIPTION
**Pull request by qmonmert**
Fix https://github.com/jhipster/generator-jhipster/pull/22134

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by qmonmert** [comment]
Error with `int` on entity
image

File: test-integration/samples/webflux-gradle-session-h2mem-es/.yo-rc.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "cacheProvider": "no",
     "clientFramework": "angular",
     "clientPackageManager": "npm",
-    "creationTimestamp": 1596513172471,
+    "creationTimestamp": 1596513172472,
```
Comment (mshima) [inline test-integration/samples/webflux-gradle-session-h2mem-es/.yo-rc.json:10]:
```suggestion
    "creationTimestamp": 1596513172473,
```

**Comment by mshima** [comment]
Failures looks related to https://github.com/jhipster/generator-jhipster/issues/20036

**Comment by mraible** [comment]
The ng-webflux-gradle-session-h2mem-es job is failing with:

```
[1]   "detail": "Unique index or primary key violation: \"PRIMARY KEY ON PUBLIC.OPERATION(ID) ( /* key:3 */ CAST(3 AS 
BIGINT), TIMESTAMP '2015-08-04 18:21:47', 'Administrator', 27987.00, NULL)\"; SQL statement:\nINSERT INTO operation 
(date, amount) VALUES ($1, $2) [23505-214]",
[1]   "instance": "/api/operations",
```

Not sure if that's related to this PR or not.

**Comment by mraible** [comment]
I generated a new microservices stack with the latest `main` branch today. I saw a lot of these warnings:

```
[@faker-js/faker]: faker.phone.phoneNumber() is deprecated since v7.3 and will be removed in v8.0. Please use faker.phone.number() instead.
```

Hopefully, this PR will fix that.

**Comment by mraible** [comment]
Any updates on this PR @qmonmert?

**Comment by mshima** [comment]
It may be related to https://github.com/r2dbc/r2dbc-h2/issues/238.

**Review by DanielFran — APPROVED** [review]

**Comment by qmonmert** [comment]
@DanielFran https://opencollective.com/generator-jhipster/expenses/148918 🙂

**Comment by DanielFran** [comment]
@qmonmert approved

**Comment by qmonmert** [comment]
thanks

**Pull request merged**